connectors/services/job_execution.py

```diff
@@ -146,11 +146,17 @@ async def _run(self):
         self.sync_job_index = SyncJobIndex(self.es_config)
 
         native_service_types = self.config.get("native_service_types", []) or []
-        logger.debug(f"Native support for {', '.join(native_service_types)}")
+        if len(native_service_types) > 0:
+            logger.debug(
+                f"Native support for job execution for {', '.join(native_service_types)}"
+            )
+        else:
```

Comment:
I think we can remove `else` here. We try to remove the native connectors concept from docs, so when customers run connector service locally, they are expected to run connector clients. 

This may confuse customers:
>No native service types configured for job execution